src/main/java/build/buildfarm/common/grpc/StubWriteOutputStream.java

```diff
@@ -259,7 +259,7 @@ public void write(int b) throws IOException {
   @Override
   public synchronized boolean isReady() {
     if (writeObserver == null) {
-      return false;
+      return true;
     }
```

Comment:
This should no longer be true, and it should be an error to call this before getOutput.